src/grid.py

```diff
@@ -6,18 +6,97 @@ class Grid:
     """
     def __init__(self, width):
 
-        if width*2 > ((width*2) + int(width/10)*4):
-            raise IndexError('height must be twice of width')
         if width%10 != 0:
             raise IndexError('width must be a multiple of 10')
 
         self.width = width
         self.height = ((width*2) + int(width/10)*4)
+        self.GRID_WIDTH = 10
+        self.GRID_HEIGHT = 24
+        self.block_width = self.width/self.GRID_WIDTH
+        self.block_height = self.height/self.GRID_HEIGHT
 
         self.grid = []
         
         for ii in range(self.width):
             self.grid.append([])
             for jj in range(self.height):
-                self.grid[ii].append(block.Block((0,0,0),self.width/10,self.height/24,(ii*(self.width/10),jj*(self.height/24))))
+                self.grid[ii].append(block.Block((0,0,0),self.block_width,self.block_height,(ii*(self.block_width),jj*(self.block_height))))
+
+    def move_down(self, screen, pos):
+        if pos != None:
+            if pos[1] < self.GRID_HEIGHT-1:
```

Comment:
combine these two lines into one `if` block: `if pos != None and pos[1] < self.GRID_HEIGHT - 1:`